VERSION

```diff
@@ -1 +1 @@
-1.5.2
+1.6.0
```

Comment:
Do we need to update the plugin version? The plugin version should not need to match the foglamp version, its' versioning can be independent of foglamp version?

Reply:
@MarkRiddoch This is something we have already discussed and agreed to.